extensions/dependencies/count_vectorizer.html

```diff
@@ -0,0 +1 @@
+<script src="{{cache_slug}}/core/templates/dev/head/domain/classifier/CountVectorizerService.js"></script>
```

Comment:
Let's combine them for now. @vojtechjelinek is looking into migrating interaction dependencies and the fewer things there are floating around, the better.